lib/src/util.dart

```diff
@@ -131,3 +131,55 @@ String escapePunctuation(String input) {
 
   return buffer.toString();
 }
+
+extension StringExtensions on String {
+  /// Removes leading whitespace by the length of [length].
+  // The way of handling tabs: https://spec.commonmark.org/0.30/#tabs
+  IndentedText indent([int length = 4]) {
```

Comment:
I think this should be called _dedent_. "indent" would _add_ whitespace.

Reply:
Fixed. thanks!